config/examples/Creality/Ender-3/BigTreeTech SKR E3-DIP V1.1/Configuration.h

```diff
@@ -473,7 +473,7 @@
 #define HEATER_5_MAXTEMP 275
 #define HEATER_6_MAXTEMP 275
 #define HEATER_7_MAXTEMP 275
-#define BED_MAXTEMP      125
+#define BED_MAXTEMP      150
```

Comment:
Really? Do these stock beds go to 150?

Reply:
Nope.

Reply:
This is the thing I called "false positive difference". The default config has the same value:
`#define BED_MAXTEMP      150`
AFAIK this parameter defines only upper margin for Bed Temperature and does not affect print quality directly. IMHO it just doesn't need to be changed.

Reply:
This should be reverted. These beds cannot go that high and will cause your printer to halt if you try.